contrib/nuke-graph-directory.sh

```diff
@@ -51,13 +51,10 @@ done
 
 # now, let's go destroy individual btrfs subvolumes, if any exist
 if command -v btrfs > /dev/null 2>&1; then
-	root="$(df "$dir" | awk 'NR>1 { print $NF }')"
-	root="${root%/}" # if root is "/", we want it to become ""
-	for subvol in $(btrfs subvolume list -o "$root/" 2>/dev/null | awk -F' path ' '{ print $2 }' | sort -r); do
-		subvolDir="$root/$subvol"
-		if dir_in_dir "$subvolDir" "$dir"; then
-			( set -x; btrfs subvolume delete "$subvolDir" )
-		fi
+	# Find btrfs subvolumes under $dir checking for inode 256
+	# Source: http://stackoverflow.com/questions/25908149/how-to-test-if-location-is-a-btrfs-subvolume
```

Comment:
Do you think it'd be better for posterity to link directly to the answer this was pulled from instead of just the question in general? (ie, http://stackoverflow.com/a/32865333/433558)